### ISSUE TITLE
Hi, have you reproduce the effect in the paper?

### ISSUE THREAD

**Issue by ares89** [daeheepark/SceneTransformer-pytorch#2]
Hi, have you reproduce the effect in the paper Scene Transformer? Thank you.

**Comment by daeheepark** [comment]
@ares89 Thank you for your interest. I have not reproduced the performance in the paper yet. The PointNet encoding for lane encoding should be additionally implemented for reproducing.

**Comment by ares89** [comment]
Thank you for your reply. I will try PointNet. 
Do you have a plan to add LaplaceKL Loss, Classification Loss and positional embedding?

**Comment by daeheepark** [comment]
@ares89 Yes, I have a plan to complete implementation, but it might be after June

**Comment by daeheepark** [comment]
@ares89 Yes, I have a plan to complete implementation, but it might be after May
